docs/examples/non_linear_pipeline_with_multiple_backends/README.md

```diff
@@ -0,0 +1,81 @@
+# Stable diffusion
+
+This example focuses on showcasing two of Triton Inference Server's features:
+* Using multiple frameworks in the same inference pipeline
+* Using the Python Backend API to build complex non linear pipelines.
+
+## How to run?
+
+Before starting, clone this repository and navigate to the root folder. It is recommended to use three different terminals for ease.
```

Comment:
Try not to infer, just state... replace the last sentence with just "Use three different terminals for an easier user experience"